dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -826,6 +827,144 @@ ShowStatusPropertyDialog(
     }
 }
 
+VOID ShowNetworkIconContextMenu(
+    _In_ HWND hwndOwner,
+    _In_ LANSTATUSUI_CONTEXT *pContext)
+{
+    if (!pContext || !pContext->pNet)
+        return;
+
+    HMENU hMenu = CreatePopupMenu();
+    if (!hMenu) return;
+
+    POINT pt;
+    GetCursorPos(&pt);
+
+    SetForegroundWindow(hwndOwner);
+
+    // The context menu items, set to their default values.
+    struct
+    {
+        UINT uID;
+        UINT uFlags;
+        UINT_PTR uIDNewItem; // wCmdID
```

Comment:
```suggestion
        UINT_PTR uIDNewItem;
```
(i forgot to remove this from my proposal.)